db/migrate/20180226101630_index_finished_sending_at_on_emails.rb

```diff
@@ -0,0 +1,7 @@
+class IndexFinishedSendingAtOnEmails < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+
+  def change
+    add_index :emails, [:finished_sending_at], algorithm: :concurrently
```

Comment:
No need for the index to be in an array here.

Reply:
Cool thanks